qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -772,6 +774,58 @@ def __getitem__(self, key):
 
         return MatrixIterator(self)
 
+    def _create_graph(self, qubit_wise):
+        """Transform measurement operator grouping problem into graph coloring problem
+
+        Returns:
+            retworkx.PyGraph: A class of undirected graphs
+        """
+
+        edges = self.paulis._noncommutation_graph(qubit_wise)
+        graph = rx.PyGraph()
+        graph.add_nodes_from(range(self.size))
+        graph.add_edges_from_no_data(edges)
+        return graph
+
+    def group_commuting(self, qubit_wise=False):
+        """Partition a SparsePauliOp into sets of commuting Pauli strings.
+
+        Args:
+            qubit_wise (bool): the commutation rule is mutually qubit-wise or not.
+
+                For example:
+
+                    * ``SparsePauliOp.from_list([('XX', 2),``
+                    * ``                               ('YY', 1),``
+                    * ``                               ("IZ",2j),``
+                    * ``                               ("ZZ",1j)]).group_commuting()``
+                            Returns [SparsePauliOp(['IZ', 'ZZ'], coeffs=[0.+2.j, 0.+1.j]),
+                                            SparsePauliOp(['XX', 'YY'], coeffs=[2.+0.j, 1.+0.j])]
```

Comment:
This won't render correctly in the documentation.  You can build the documentation with `tox -edocs` to test.  I imagine you mean something like
```text
Args:
    qubit_wise (bool): whether the commutation rule is applied to the whole operator,
        or on a per-qubit basis.  For example:

        .. code-block:: python

            >>> op = SparsePauliOp.from_list([("XX", 2), ("YY", 1), ...])
            >>> op.group_commuting()
            [SparsePauliOp(["IZ", "ZZ"], coeffs=[0.+2.j, 0.+1j]),
             SparsePauliOp(["XX", "YY"], coeffs=[2.+0.j, 1.+0.j])]
            >>> op.group_commuting(qubit_wise=True)
            [...]
```

Reply:
Thank you for instructing documentation syntax. This expression and test command are exactly what I sought for.
So far, I will adopt your suggested wording of `qubit_wise` argument and be consistent with that wording of dosctrings over methods which use the argument.